corehq/motech/openmrs/repeaters.py

```diff
@@ -55,16 +64,31 @@ def get_custom_url(cls, domain):
         from corehq.motech.repeaters.views.repeaters import AddOpenmrsRepeaterView
         return reverse(AddOpenmrsRepeaterView.urlname, args=[domain])
 
+    def get_location(self):
+        if self.location_id:
+            try:
+                return SQLLocation.objects.get(location_id=self.location_id)
```

Comment:
Why not use `SQLLocation.by_location_id`?

Reply:
Good call.

Reply:
Ha. Turns out I use `repeater.location_id` in but never `repeater.get_location()`. Dropping this because YAGNI.